addons/hr_timesheet/migrations/10.0.1.0/pre-migration.py

```diff
@@ -11,6 +11,25 @@
 }
 
 
+def create_and_populate_department(cr):
+    cr.execute('''
+       ALTER TABLE account_analytic_line ADD COLUMN department_id INT;
+       ALTER TABLE account_analytic_line
```

Comment:
Isn't constraint added automatically in the update process? It's risky to replicate ORM in this.

Reply:
right, this was a bit premature. No manual constraints, they'll be recreated also when the column exists, cf https://github.com/OCA/OCB/blob/11.0/odoo/models.py#L2175 - @mvaled could you PR a followup for this?